python/CHANGELOG.rst

```diff
@@ -59,6 +60,24 @@
   which assigns a string to node and mutation symbols, commonly shown on mouseover. This
   can reduce label clutter while retaining useful info (:user:`hyanwong`, :pr:`3007`)
 
+- Added ``distance_between`` that calculates the total distance between two nodes in a tree.
+  (:user:`Billyzhang1229`, :pr:`2771`)
+
+- Added ``genetic_relatedness_matrix`` method to compute
+  pairwise genetic relatedness between sample sets.
+  (:user:`jeromekelleher`, :user:`mmosmond`, :user:`petrelharp`, :pr:`2823`, :pr:`1623`)
+
+- Added ``genetic_relatedness_vector`` method to compute product of genetic relatedness
+  matrix and weight vector.
+  (:user:`petrelharp`, :pr:`2980`)
+
+- Added ``pair_coalescence_counts`` method to calculate coalescence events per node or time
+  interval, and ``pair_coalescence_quantiles`` method to estimate quantiles of pair
+  coalescence times using empirical CDF inversion.
```

Comment:
also:

and ``pair_coalescence_rates`` method to estimate instantaneous rates of pair coalescence within time intervals from the empirical CDF (:pr:`2985`)